extensions/2.0/Khronos/KHR_materials_iridescence/README.md

```diff
@@ -0,0 +1,148 @@
+|**iridescenceFactor**         | `number`                                                            | The iridescence intensity.                                                                       | No, default: `0.0`    |
+|**iridescenceTexture**        | [`textureInfo`](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo) | The iridescence intensity stored in the red channel of the texture.                                                               | No                    |
+|**iridescenceIOR**               | `number`                                                            | The index of refraction of the dielectric thin-film layer.                                       | No, default: `1.8`    |
+|**iridescenceThicknessMinimum**  | `number`                                                            | The minimum thickness of the thin-film layer given in nanometers (nm).                                                    | No, default: `400.0`  |
+|**iridescenceThicknessMaximum**  | `number`                                                            | The maximum thickness of the thin-film layer given in nanometers (nm).                                                    | No, default: `1200.0` |
+|**iridescenceThicknessTexture**  | [`textureInfo`](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo) | The green channel of this texture defines the thickness of the thin-film layer by blending between the minimum and maximum thickness. | No                    |
+
+The values for iridescence intensity can be defined using a factor, a texture, or both.
+`iridescenceFactor` is multiplied with the red channel of `iridescenceTexture` to control the overall strength of the iridescence effect. If the texture is not set, a value of 1.0 is assumed. 
+```
+iridescence = iridescenceFactor * iridescenceTexture.r
+```
+
+If `iridescenceFactor` is zero (default), the iridescence extension has no effect on the material. 
+All textures in this extension use a single channel in linear space. 
+The thickness of the thin-film is set to `iridescenceThicknessMaximum` if `iridescenceThicknessTexture` is not given.
+If `iridescenceThicknessTexture` is set, the thickness of the thin-film varies between  `iridescenceThicknessMinimum` and `iridescenceThicknessMaximum` as follows: 
+```
+thickness = mix(iridescenceThicknessMinimum, iridescenceThicknessMaximum, iridescenceThicknessTexture.g)
+```
+The thin-film layer can have a different IOR than the underlying material. With `iridescenceIOR` one can set an IOR value for the thin-film layer independently.
+
+The iridescence effect is modeled via a microfacet BRDF with a modified Fresnel reflectance term that accounts for inter-reflections as shown in [Laurent Belcour and Pascal Barla (2017)](https://belcour.github.io/blog/research/publication/2017/05/01/brdf-thin-film.html).
+
```

Comment:
In the sheen extension there is a [long, non-normative section "Implementation Notes"](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_sheen/README.md#implementation-notes) that gives a sample implementation of the sheen effect. I like it a lot, because I think it is a good starting point for a custom implementation. Do you plan to add such a section to this extension?

Reply:
Good point. @UX3D-becher

Reply:
Tobias, feel free to add things and add your name to the creators of this extension.